extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -381,13 +422,20 @@ export class AmpA4A extends AMP.BaseElement {
             verifier.loadKeyset(signingServiceName);
           });
         });
+
+    if (this.a4aAnalyticsConfig_) {
```

Comment:
Let's delay initializiation of a4aAnalyticsConfig_ until here so that it happens in buildCallback instead of constructor

Reply:
Done